datarepo-clienttests/src/main/resources/configs/BulkLoadScale200_100.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "BulkLoadScale200_100",
+  "description": "One user ingests many files into one dataset",
+  "serverSpecificationFile": "localhost.json",
+  "billingAccount": "00708C-45D19D-27AAFA",
+  "kubernetes": {
+    "numberOfInitialPods" : 1
+  },
+  "application": {
+    "maxStairwayThreads" : 120,
+    "loadConcurrentFiles": 100,
```

Comment:
Good idea. Looking at the code, I think that require a good bit of work, but definitely something to think about for future work. 
Tagging @marikomedlock to keep her in the loop!

Reply:
Yes, I discussed this with Dan also. More parameterization is more convenient for developing/writing tests, but I think it's less clear when we're running tests to figure out what was actually run. Anyway, I think this might be a good quality of life improvement in the future, but a low priority for now. One way to do this is to leave the TestConfig alone -- I think we should keep the idea that 1 TestConfig corresponds to 1 test run -- and do something at the TestSuite level.